sys-apps/sum-driver/files/makefile.patch

```diff
@@ -0,0 +1,56 @@
+--- a/Makefile	2015-12-01 05:16:51.000000000 +0100
```

Comment:
Where does this come from? Why it's a patch when it changes every single line in the file?

Reply:
I rewrote the complete makefile. I changed it to a file to copy under FILESDIR